public/docs/ts/latest/cookbook/a1-a2-quick-reference.jade

```diff
@@ -584,21 +582,21 @@ table(width="100%")
       code-example.
         &lt;tr ng-repeat="movie in movieList | orderBy : 'title'">
       :marked
-        Orders the collection, as specified by the expression.
+        `OrderBy` orders the collection, as specified by the expression.
```

Comment:
Good point. From my limited experience in Angular, I've seen orderBy used for the default way the data is displayed and sorting (coded in a variety of ways) used for user manipulation after the data has been rendered in its default way. With that in mind, how about this:

Displays the collection...